app/src/main/java/org/simple/clinic/navigation/v2/fragments/BaseScreen.kt

```diff
@@ -79,7 +79,11 @@ abstract class BaseScreen<K : ScreenKey, B: ViewBinding, M : Parcelable, E, F, R
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
 
-    controller.connect(Connectables.contramap({ it }, ::connectViews))
+    val uiRenderer = uiRenderer()
+    controller.connect(
+        Connectables.contramap({ it })
```

Comment:
Since we are not modifying the model before rendering, should be stop using `contramap` now?

Reply:
Yeah, this makes sense. I'll do this.